crates/reporting/tests/test_reporting.rs

```diff
@@ -55,6 +58,38 @@ mod test_reporting {
         buffer
     }
 
+    fn maybe_save_parse_test_case(test_name: &str, src: &str, is_expr: bool) {
+        // First check if the env var indicates we should migrate tests
+        if std::env::var("ROC_MIGRATE_REPORTING_TESTS").is_err() {
+            return;
+        }
+
+        // Check if we have parse errors
+        let arena = Bump::new();
+        let has_error = if is_expr {
+            parse_expr_with(&arena, src).is_err()
+        } else {
+            parse_header(&arena, State::new(src.trim().as_bytes())).is_err()
+            // TODO: also parse the module defs
+        };
+
+        if !has_error {
+            return;
+        }
+
+        let mut path = std::env::current_dir().unwrap();
+        assert!(path.pop());
+        path.push("compiler");
+        path.push("parse");
+        path.push("tests");
+        path.push("snapshots");
+        path.push("fail");
```

Comment:
consider defining instead

```
let mut path = std::concat!(std::env!("ROC_WORKSPACE_DIR"), "/crates/compiler/parse/tests/snapshots/fail).into();
```

The workspace env var is defined in https://github.com/roc-lang/roc/blob/main/.cargo/config

Reply:
Will the forward slashes work on windows?

Reply:
thats a good point, push works too. My comment is more towards defining the path relative to the root of the repo rather than relative to the test file, since it touches different packages and the former method will mean this diff doesn't have to be affected if the file moves.